satpy/tests/test_resample.py

```diff
@@ -249,114 +249,6 @@ def test_check_numpy_cache(self, xr_Dataset, np_load):
             shutil.rmtree(the_dir)
 
 
-class TestEWAResampler(unittest.TestCase):
```

Comment:
Don't you want to keep this while the legacy version is available?

Reply:
I can't. If the new pyresample version is released then these tests fail because they use some mocking (and maybe kwargs) that don't apply to the new algorithm.

Reply:
`@pytest.mark.skip_if` ?

Reply:
Sounds good.

Reply:
Done.